models/src/main/java/io/sphere/sdk/attributes/AttributeContainerImpl.java

```diff
@@ -0,0 +1,50 @@
+package io.sphere.sdk.attributes;
+
+import io.sphere.sdk.json.JsonException;
+import io.sphere.sdk.models.Base;
+import io.sphere.sdk.products.Product;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import static java.lang.String.format;
+
+public class AttributeContainerImpl extends Base implements AttributeContainer {
```

Comment:
This class should not (yet) be public. You can move it to the package and make it package scope.